packages/sw-cli/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "sw-cli",
   "version": "0.0.8",
-  "private": true,
```

Comment:
Lerna will make them all publish with same version if they change and when they do - they jump to the latest version number. I think this has been good so far and makes sense to avoid needless churn for updates for projects depending on us.